Modules/QtWidgets/src/QmitkStdMultiWidget.cpp

```diff
@@ -225,7 +225,11 @@ void QmitkStdMultiWidget::ResetCrosshair()
     return;
   }
 
-  mitk::RenderingManager::GetInstance()->InitializeViewsByBoundingObjects(dataStorage);
+	auto geo = dataStorage->ComputeBoundingGeometry3D(dataStorage->GetAll());
+	mitk::RenderingManager::GetInstance()->InitializeView(GetRenderWindow1()->GetRenderWindow(), geo);
+	mitk::RenderingManager::GetInstance()->InitializeView(GetRenderWindow2()->GetRenderWindow(), geo);
+	mitk::RenderingManager::GetInstance()->InitializeView(GetRenderWindow3()->GetRenderWindow(), geo);
+	mitk::RenderingManager::GetInstance()->InitializeView(GetRenderWindow4()->GetRenderWindow(), geo);
```

Comment:
If we are displaying `QmitkStdMultiWidget` and `QmitkRenderWindow` both at the same time, each one rendering a different data set, `mitk::RenderingManager::GetInstance()->InitializeViewsByBoundingObjects(dataStorage);` would set the geometry from `datastorage` rendered by `QmitkStdMultiWidget` to `QmitkRenderWindow`, the last one may contain different data with different geometries. This is not a issue in MitkWorkbench because, usually, `QmitkStdMultiWidget` is the only active render widget.

Reply:
So you are explicitly initializing only the four render windows of the `QmitkStdMultiWidget` so that other render windows (that are registered with the global rendering manager) are not affected here.
I think that is reasonable and makes sense here.